test/runtests.jl

```diff
@@ -909,18 +909,47 @@ else  # run HiGHS tests
         end
 
         @testset "Minimize Unserved Load" begin
-                
+            d = JSON.parsefile("./scenarios/outage.json")
+            d["ElectricLoad"]["loads_kw"] = ones(8760)*200.0
+            d["ElectricLoad"]["loads_kw"][5100:5109] .= 410.0
+            d["ElectricLoad"]["loads_kw"][5200:5209] .= 410.0
+            d["ElectricLoad"]["critical_load_fraction"] = 0.5
+            d["PV"]["existing_kw"] = 0.0
+            d["PV"]["min_kw"] = 100.0
+            d["PV"]["max_kw"] = 100.0
+            d["CHP"]["min_kw"] = 100.0
+            d["CHP"]["max_kw"] = 100.0
+            d["Generator"]["existing_kw"] = 0.0
+            d["Generator"]["min_kw"] = 100.0
+            d["Generator"]["max_kw"] = 100.0
+            d["ElectricStorage"]["min_kw"] = 20
+            d["ElectricStorage"]["max_kw"] = 20
+            d["ElectricStorage"]["min_kwh"] = 50
+            d["ElectricStorage"]["max_kwh"] = 50
+            d["Financial"]["microgrid_upgrade_cost_fraction"] = 0.0
+            s = Scenario(d)
+            p = REoptInputs(s)
             m = Model(optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => false, "log_to_console" => false, "mip_rel_gap" => 0.01, "presolve" => "on"))
-            results = run_reopt(m, "./scenarios/outage.json")
-
-            @test results["Outages"]["expected_outage_cost"] ≈ 0
-            @test sum(results["Outages"]["unserved_load_per_outage_kwh"]) ≈ 0
+            results = run_reopt(m, p)
+        
+            @test results["Outages"]["expected_outage_cost"] ≈ 0 atol=0.1
+            @test sum(results["Outages"]["unserved_load_per_outage_kwh"]) ≈ 0 atol=0.1
             @test value(m[:binMGTechUsed]["Generator"]) ≈ 1
             @test value(m[:binMGTechUsed]["CHP"]) ≈ 1
             @test value(m[:binMGTechUsed]["PV"]) ≈ 1
             @test value(m[:binMGStorageUsed]) ≈ 1
-            @test results["Financial"]["lcc"] ≈ 6.83633907986e7 rtol=0.01
-
+        
+            # Increase cost of microgrid upgrade and PV Size, PV not used and some load not met
+            d["Financial"]["microgrid_upgrade_cost_fraction"] = 0.3
+            d["PV"]["min_kw"] = 200.0
+            d["PV"]["max_kw"] = 200.0
+            s = Scenario(d)
+            p = REoptInputs(s)
+            m = Model(optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => false, "log_to_console" => false, "mip_rel_gap" => 0.01, "presolve" => "on"))
+            results = run_reopt(m, p)
+            @test value(m[:binMGTechUsed]["PV"]) ≈ 0
+            @test sum(results["Outages"]["unserved_load_per_outage_kwh"]) > 0
```

Comment:
@zolanaj any reason to not set this to the value of 24.16 instead of just > 0 ?

Reply:
I think this one is stable enough to make 24.16.  I usually stay away from fixed values when a 1% mip tolerance is involved, but I think this one should be ok fo now.  Good catch!

Reply:
done in 296bbd63acc4755283536de6c70c88e191ad54f7